### PR TITLE
[Filter/Custom] copy metadata in registration

### PR DESCRIPTION
**Pull request by jaeyun-jung**
In the filter custom-easy, prevent error case when developer tries to register custom-filter with local variables.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c [hunk 1]
```diff
@@ -66,15 +66,19 @@ NNS_custom_easy_register (const char *modelname,
...
 
   if (!ptr)
     return -ENOMEM;
 
   ptr->func = func;
   ptr->data = data;
-  ptr->in_info = in_info;
-  ptr->out_info = out_info;
+  gst_tensors_info_copy (&ptr->in_info, in_info);
+  gst_tensors_info_copy (&ptr->out_info, out_info);
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c:81]: I think you need to call `gst_tensors_info_free` as well in error handling (e.g., before `g_free (ptr)` in line 86).

File: gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c [hunk 2]
```diff
@@ -66,15 +66,19 @@ NNS_custom_easy_register (const char *modelname,
   if (!func || !in_info || !out_info)
     return -EINVAL;
 
+  if (!gst_tensors_info_validate (in_info) ||
```
Comment (gichan-jang) [inline gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c:69]: need to pass the pointer of the info.

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c:69]: in_info is a pointer :)

File: gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c (hunk 1 above)
Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c:93]: You are right. I added a function to free internal data.
When unregistering custom filter, it is also necessary to free internal data.

File: gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c (hunk 2 above)
Comment (gichan-jang) [reply gst/nnstreamer/tensor_filter/tensor_filter_custom_easy.c:81]: Ah, I confused with line 41. Thanks.

**Pull request merged**